src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -25,25 +25,17 @@
 public class AddCommand extends Command implements ReversibleCommand {
 
     public static final String COMMAND_WORD = "add";
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a person to the address book. \n"
-            + "Parameters: "
-            + PREFIX_NAME + "NAME "
-            + PREFIX_PHONE + "PHONE "
-            + PREFIX_EMAIL + "EMAIL "
-            + PREFIX_CATEGORY + "CATEGORY "
-            + "[" + PREFIX_GROUP + "GROUP]\n"
-            + "Example: " + COMMAND_WORD + " "
-            + PREFIX_NAME + "John Doe "
-            + PREFIX_PHONE + "98765432 "
-            + PREFIX_EMAIL + "johnd@example.com "
-            + PREFIX_CATEGORY + "participant "
-            + PREFIX_GROUP + "3 ";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a person to the contact list. \n"
```

Comment:
It looks better to use original format for readability

Reply:
Updated, please review